README.md

```diff
@@ -1,40 +1,55 @@
 # Team Selector
 
-I need to make student teams in my classes for which I ask students to specify some people (up to 3) that they want to pair with and some (up to 2) that they _do not_ want to work with. This program outputs candidate pairings which are the most in accordance with the students' preferences.
+I need to make student teams of 2 in my classes for which I ask students to specify _greens_ and _reds_. Greens are people that the student wants to work with and reds are people that they _do not_ want to work with. This program matches students according to these prefernces.
 
 ## How to Use
 
-You _only_ need to make the following edits in the `init_students` function toward the bottom of the file,  _select.py_.
-1. Add the names of _all_ the students in your class who need to be paired to the indicated list. This includes students who did not indicate any preference, e.g. because they did not fill the required form by the deadline, but still need to be paired. The names _must be unique_.
-1. Enter the preferences using the `add_green` and `add_red` functions. Their syntax is as follows.
+You _only_ need to make the filloiwng edits in the `enter_data` function toward the bottom of the file,  _match.py_. The segments to be edited are clearly marked in the file with __EDIT__.
```

Comment:
following*

Reply:
Did a spell check and rewrote the section with instructions for the CSV file.